helm-charts/iqserver/templates/configmap.yaml

```diff
@@ -1,7 +1,18 @@
+---
 apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ template "iqserver.fullname" . }}
 data:
   config.yml: |
 {{ toYaml .Values.iq.configYaml | indent 4 }}
+
+{{ if .Values.iq.certificates }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "iqserver.fullname" . }}-certificates
+data:
+{{ toYaml .Values.iq.certificates | indent 2 }}
+{{- end }}
```

Comment:
if `iq.certificates` are provided in values.yaml, create a whole config map that will be mounted into the init container for processing.